electrum/bitcoin.py

```diff
@@ -752,6 +753,36 @@ def is_private_key(key: str, *, raise_on_error=False) -> bool:
             raise
         return False
 
+def parse_script(x):
+    script = ''
+    for word in x.split():
+        if word[0:3] == 'OP_':
+            opcode_int = opcodes[word]
+            script += construct_script([opcode_int])
+        else:
+            bfh(word)  # to test it is hex data
+            script += construct_script([word])
+    return script
+
+def parse_address(line):
+    r = line.strip()
+    m = re.match('^'+RE_ALIAS+'$', r)
+    address = str(m.group(2) if m else r)
+    assert is_address(address)
+    return address
+
+def parse_output(x) -> bytes:
+    try:
+        address = parse_address(x)
+        return bfh(address_to_script(address))
+    except Exception:
+        pass
```

Comment:
I would strongly prefer not exposing the openalias regex stuff to the CLI, and would keep that in the Qt GUI.
First, I suggest doing this here instead:

```suggestion
    if is_address(x):
        return return bfh(address_to_script(address))
```

Second, let's keep `parse_address` in the GUI code where it was.
Then, to maintain the current behaviour for the GUI code, also keep a method `parse_output` in the GUI, which first calls `parse_address` and then falls back to this methiod here (`bitcoin.parse_output`).